thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/alertFilterAutotune/DummyAlertFilterAutoTune.java

```diff
@@ -9,6 +9,13 @@
 public class DummyAlertFilterAutoTune implements AlertFilterAutoTune {
 
   @Override
+  public Map<String, String> initiateAutoTune(List<MergedAnomalyResultDTO> anomalyResults, int nExpectedAnomalies)
+      throws Exception {
+    Map<String, String> emptyMap = new HashMap<>();
+    return emptyMap;
```

Comment:
If the map is not going to be modified, you could return Collections.emptyMap().